app/models/distribution.rb

```diff
@@ -133,4 +133,8 @@ def csv_export_attributes
       line_items.total
     ]
   end
+
+  def past?
```

Comment:
Sorry, totally my bad for not catching this earlier, but if the inequality goes this way we should name this function future?

Reply:
It makes sense. I didn't think about this either. Changing...